### DISCUSSION TITLE
Privacy policy statement /GDPR compliance

### DISCUSSION THREAD

**Discussion by maraunzer12** [ImageMagick/ImageMagick#7133] [General]
Hi all,
i have been a happy user of imagemagick, and I wanted to promote the use of imagemagick in  my company as well.

Unfortunately, due to privacy policies  we need to consider for legal reasons, that is not compliant.

I understand that we could just not use imagemagick, but if the privacy policy could be updated to match regulations such as GDPR it would be of  great help.

Some examples could be Mozilla or Notepad++ privacy policies

Thank you in advance for support

**Comment by dlemstra** [comment]
Can you explain in more detail what the issue is? Our software does not collect any data of the user.

**Comment by maraunzer12** [comment]
i cannot find on privacy policy page that is expressly specified that are not data collected , could you help me to verify this point  ( i was pointing on security section) ?
Also the same is related to tracker ,ads etc ?
If you could  send me the reference link will be great ,

Thank you in advance

**Reply by dlemstra** [reply]
Are you mixing the privacy policy of our website with what our application/library is doing? I am not a lawyer so I don't understand why our website privacy policy would be an issue when you want to use our application/library?

**Reply by maraunzer12** [reply]
Thank for the reply,
my concern is about that i cannot found  that the data , cookie or tracker are not collected , is possible to mention in the page ?
https://imagemagick.org/script/security-policy.php

**Reply by dlemstra** [reply]
That is our security policy and not a privacy policy. I really don't understand what you want and why you need it. Please give more details.

**Reply by maraunzer12** [reply]
ok could you provide me the privacy policy link ? i cannot find on website

**Comment by urban-warrior** [comment]
GDPR imposes obligations on organizations targeting or collecting data related to people in the EU. As the organization overseeing the open-source ImageMagick software distribution, ImageMagick Studio LLC does not collect any personal identifier information (PII).  In addition to the organization, the ImageMagick software distribution does not collect PII either.  Refer to https://imagemagick.org/script/privacy-policy.php.
